pkgs/tools/text/bashblog/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchzip, fetchFromGitHub, makeWrapper, bash, pandoc }:
+
+stdenv.mkDerivation rec {
+  pname = "bashblog";
+  version = "unstable-2022-03-26";
+
+  src = fetchFromGitHub {
+    owner = "cfenollosa";
+    repo = "bashblog";
+    rev = "c3d4cc1d905560ecfefce911c319469f7a7ff8a8";
+    sha256 = "sha256-THlP/JuaZzDq9QctidwLRiUVFxRhGNhRKleWbQiqsgg=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildInputs = [ pandoc ];
+
+  patches = [
+    ./0001-Setting-markdown_bin.patch
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
```

Comment:
You should do your patches before you install.

Also, add `patchShebangs` to your inputs and add `patchShebangs bb.sh`, so that you don't need `makeWrapper`.

Reply:
Didn't know about `patchShebangs`. It seems `makeWrapper ` already has it, so it was just applying it in the `postPatch`.